src/deps/c_ares.zig

```diff
@@ -773,6 +773,19 @@ fn libraryInit() void {
     if (ares_has_loaded.swap(true, .monotonic))
         return;
 
+    if (!bun.use_mimalloc) {
```

Comment:
if we're not using mimalloc we can skip calling this function,as it uses the c allocator by default